src/routes/dashboard/index.svelte

```diff
@@ -1,250 +1,244 @@
-            class="page-link-card create-standup-link-card"
-          >
-            <img
-              src="create-standup-card-img.svg"
-              alt=""
-            />
-            <p>Create Standup</p>
-          </a>
-          <a
-            href="/dashboard/standups/"
-            class="page-link-card all-standups-link-card"
-          >
-            <img src="all-standups-card-img.svg" alt="" />
-            <p>All Standups</p>
-          </a>
-        </div>
-        <div class="page-link-card-row">
-          <a
-            href="/dashboard/standups/reports"
-            class="page-link-card my-standups-link-card"
-          >
-            <img src="my-standups-card-img.svg" alt="" />
-            <p>Reports</p>
-          </a>
-          <a
-            href="/dashboard/account"
-            class="page-link-card my-profile-link-card"
-          >
-            <img src="my-profile-card-img.svg" alt="" />
-            <p>My Profile</p>
-          </a>
-        </div>
-        <div class="page-link-card-row">
-          <a href="./settings.html" class="settings-link-card">
-            <img src="gear-icon.svg" alt="" />
-            <p>Settings</p>
-          </a>
-        </div>
-      </div>
-      <div class="some-stats-container">
-        <p class="text-primary">Some Stats</p>
-        <div class="stats-card">
-          <div class="header">
-            <p>Standups report</p>
-            <p class="text-primary icon-right">
-              This Week
-              <img
-                src="icons/icon-chevron-down.svg"
-                alt=""
-              />
-            </p>
-          </div>
-          <div class="body">
-            <div class="progress-rings">
-              <div class="filled-progress-circle">
-                <div class="created-progress-circle"></div>
-              </div>
-            </div>
-            <div class="progress-infos">
-              <div class="u-margin-bottom-large">
-                <p class="color-indicator color-indicator-1">&nbsp;</p>
-                <div>
-                  <p>Filled</p>
-                  <p>22</p>
-                </div>
-              </div>
-              <div>
-                <p class="color-indicator color-indicator-2">&nbsp;</p>
-                <div>
-                  <p>Created</p>
-                  <p>3</p>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div class="stats-card">
-          <div class="header">
-            <p>User activity</p>
-            <p class="text-primary icon-right">
-              This Week
-              <img
-                src="icons/icon-chevron-down.svg"
-                alt=""
-              />
-            </p>
-          </div>
-          <div class="body">
-            <div>
-              <img src="barchart.svg" alt="" />
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
+	<p class="welcome-message">
```

Comment:
Avoid adding formatting changes to code that is not directly touched. Best for new PRs